collections/_posts/2023-03-14-Deep-Label-Fusion.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: review
+title: "Deep label fusion: A generalizable hybrid multi-atlas and deep convolutional neural network for medical image segmentation"
+tags: deep-learning CNN transformer segmentation classification object-detection attention
+author: "Emile Saillard"
+cite:
+    authors: "Long Xie, Laura E.M. Wisse, Jiancong Wang, Sadhana Ravikumar, Pulkit Khandelwal, Trevor Glenn, Anica Luther, Sydney Lim, David A. Wolk, Paul A. Yushkevich"
+    title:   "Deep label fusion: A generalizable hybrid multi-atlas and deep convolutional neural network for medical image segmentation"
+    venue: Elsevier
```

Comment:
The venue is the journal or conference, not the publisher. In this case it should be _Medical Image Analysis_:
```suggestion
    venue: Medical Image Analysis (2023)
```